CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+0.4.6 (June 2019)
+-----------------
+- fixed bug in Image.getPixelValues() method which returned
+  distorted array of pixel values for images with non-square
+  dimensions
```

Comment:
Is it worth mentioning https://github.com/ome/rOMERO-gateway/pull/68 in the notes as well or are you trying to keep this changelog mostly focused on user-facing improvements?

Reply:
Was thinking about it too. But it doesn't really have anything to do with `romero.gateway` package itself, so not necessary to mention it in the changelog I think.

Reply:
It is still a useful feature to mention so people know it is there if they want to try it out without having to install anything locally